src/main/java/com/airlines/dao/impl/AirplaneDaoImpl.java

```diff
@@ -103,7 +128,7 @@ public void remove(Airplane airplane) {
             throw new DaoOperationException(String.format("Product with id = %d does not exist", airplane.getId()));
         }
         try (Connection connection = dataSource.getConnection();
-             PreparedStatement statement = connection.prepareStatement(DELETE_AIRPLANE_SQL);) {
+             PreparedStatement statement = connection.prepareStatement(DELETE_AIRPLANE_SQL)) {
```

Comment:
DaoImpl is not part of your task
Discard this change

Reply:
done in https://github.com/Matsukata/airlines-jdbc-app/pull/5/commits/625cab1de0c97028d188cc5a80294e281f46b375